single-product.html

```diff
@@ -0,0 +1,673 @@
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">  <strong class="star-symbol">✰✰</strong></label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">  <strong class="star-symbol">✰</strong></label>
+                            </div>
+                        </a>
+                    </li>
+                </ul>
+                <ul class="ul">
+                    <h2 class="sidebar-cmp-heading">
+                        Price
+                    </h2>
+                    <div class="slider-container">
+                        <input id="range-slider" class="slider" type="range" name="slider" value="10" min="0"
+                            max="5000">
+                        <p id="range-value">0</p>
+                    </div>
+                </ul>
+                <ul class="ul">
+                    <h2 class="sidebar-cmp-heading">Discount</h2>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">10% Off or
+                                    more</label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">25% Off or
+                                    more</label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">35% Off or
+                                    more</label>
+                            </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="link" href="#">
+                            <div class="checkbox-input">
+                                <input id="check" type="checkbox" name="check" /><label for="check">50% Off or
+                                    more</label>
+                            </div>
+                        </a>
+                    </li>
```

Comment:
Change:- The ID and "for" attributes are the same here for everyone because of which clicking on the label of any of these will tick the first one. So make sure to give different IDs and 'for' of the different checkboxes.

Reply:
Will do the changes asap. :)